src/main/java/com/synopsys/integration/alert/channel/jira/server/web/JiraServerCustomEndpoint.java

```diff
@@ -80,7 +80,7 @@ public JiraServerCustomEndpoint(JiraServerChannelKey jiraChannelKey, CustomEndpo
             String username = jiraProperties.getUsername();
             String password = jiraProperties.getPassword();
             Response response = jiraAppService.installMarketplaceServerApp(JiraConstants.JIRA_APP_KEY, username, password);
-            if (response.isStatusCodeError()) {
+            if (BooleanUtils.isTrue(response.isStatusCodeError())) {
```

Comment:
Does this have to do with using `Boolean` rather than `boolean`? If so, we should fix that instead.

Reply:
We would need to fix Response in integration-rest to resolve this. Which we should but until that gets fixed, this resolves any null issues that would come up. The current implementation of `response.isStatusCodeError()` does allow returning a null.